compute/quickstart/compute_quickstart_sample.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+const (
+	mongoport = "80"
+	port      = "80"
+)
+
+var tm mongoManager
+
+func main() {
+	var err error
+	mongohost := os.Getenv("HOST")
+
+	tm, err = newTrainerManager(mongohost, mongoport)
+	if err != nil {
+		log.Fatal(fmt.Printf("error connecting to mongo: %s", err))
+	}
+
+	trainers := []trainer{
+		{"Ash", 10, "Pallet Town"},
+		{"Misty", 10, "Cerulean City"},
+		{"Brock", 15, "Pewter City"},
+	}
+
+	if err := tm.load(trainers); err != nil {
+		log.Fatal(err)
+	}
+
+	http.HandleFunc("/", listHandler)
+	log.Fatal(http.ListenAndServe(":"+port, nil))
+}
+
+func listHandler(w http.ResponseWriter, r *http.Request) {
+	trainers, err := tm.list()
+	if err != nil {
+		httpResponse(w, http.StatusInternalServerError, []byte(err.Error()))
```

Comment:
This `httpResponse` function is so short, it's probably better to just inline it where it's used.

Reply:
Done. /